backend/src/ee/services/permission/project-permission.ts

```diff
@@ -288,18 +346,78 @@ export const ProjectPermissionSchema = z.discriminatedUnion("subject", [
       "Describe what action an entity can take."
     )
   }),
+  z.object({
+    subject: z.literal(ProjectPermissionSub.Cmek).describe("The entity this permission pertains to."),
+    inverted: z.boolean().optional().describe("Whether rule allows or forbids."),
```

Comment:
For inverted, let's sort for the user so that they don't have to think much about the inner workings of CSAL